bloom-housing/bloom
# Prisma change over feature branch 23 01 29

# Pull Request Template

## Issue Overview

This PR addresses #issue


## Description

This pulls the following commits from main:
* 42cd7fe52 - (origin/main) fix: add divider width (#3841) <ColinBuyck>
* 99847f07c - feat: 422/updated account theming (#3791) <ColinBuyck>
* 3ea47eda1 - (main) fix: Paper Application not saving lat long (#3828) <Krzysztof Zięcina>
* 09612429e - fix: add submit to sms mfa flow (#3838) <Morgan Ludtke>
* e3305e413 - feat: 3761/direct to sign in (#3816)  <cade-exygy>
* c3a6b9c67 - fix: geocoding verification fix (#3832) <Morgan Ludtke>
* 27c13b6e6 - fix: add default value for map layer (#3829) <Morgan Ludtke>
* 6fe537422 - fix: add map layer geocoding checks (#3825) <Morgan Ludtke>
* aa8e3a647 - feat: geocoding preference set up fixed layers (#3808) <Krzysztof Zięcina>
* 66816871e - fix: seeds button layout fixes for Partners Application sidebar (#3789) <Jared White>
* af18a9bb8 - fix: add links to preference description (#3813) <Krzysztof Zięcina>
* 87288ef56 - fix: provides additional fields info text overflows (#3809) <Krzysztof Zięcina>

Also implements the remaining portions of geocoding

## How Can This Be Tested/Reviewed?

The main thing that needs to be tested is the geocoding functionality. This can be tested by reseeding with stage data and applying to the "Elm Village" property for both public and the partner site. It should add the geocoding verified value on the export.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
